File: iniciante/javascript/b1015.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

let line1 = lines.shift().split(' ');
let x1 = parseFloat(line1[0]);
let y1 = parseFloat(line1[1]);
let line2 = lines.shift().split(' ');
let x2 = parseFloat(line2[0]);
let y2 = parseFloat(line2[1]);

let distancia = Math.sqrt(Math.pow(x2 - x1, 2) + Math.pow(y2 - y1, 2));

console.log(distancia.toFixed(4));

File: iniciante/swift/b1015.swift
import Foundation

var values = readLine()!.split(separator: " ")
let x1 = Double(values[0])!
let y1 = Double(values[1])!

values = readLine()!.split(separator: " ")
let x2 = Double(values[0])!
let y2 = Double(values[1])!

var distance = sqrt(pow(x2 - x1, 2) + pow(y2 - y1, 2))

print(String(format: "%.4f", distance))

